src/test/java/StringTest.java

```diff
@@ -0,0 +1,44 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class StringTest {
+
+    @DisplayName("요구사항1 - split 테스트")
+    @Test
+    void testSplit() {
+        String value = "1,2";
+        String[] split = value.split(",");
+        assertThat(split).hasSize(2);
+        assertThat(split).contains("1", "2");
+        assertThat(split).containsExactly("1", "2");
```

Comment:
중복되는 형태의 검증 메서드가 호출되어지고 있는 것으로 보이는데요.
한 가지만 호출해주셔도 좋습니다. 이유는 내부 검증 메서드의 구현 내용을 보게 되면 사이즈 체크, 내용물 확인, 순서까지 검증을 하고 있습니다.
```suggestion
        assertThat(split).containsExactly("1", "2");
```

검증문의 책임 결합도를 높이기 위해, assertAll을 활용해서 검증 메서드를 호출 해보시는 것을 추천 드립니다. 😄 
링크는 참조 하겠습니다. [Junit5 assertAll examples](https://www.javaguides.net/2018/09/junit-5-assertall-example.html)